### PR TITLE
Improve some FR translations

### PR DESCRIPTION
**Pull request by olivroy**
### Intent

Improve some FR translations. This is not exhaustive, but thought I`d edit some

### Approach

Update strings where appropriate 
### Automated Tests

N/A

### QA Notes

> Add additional information for QA on how to validate the change, paying special attention to the level of risk, adjacent areas that could be affected by the change, and any important contextual information not present in the linked issues. 
N/A

### Documentation

N/A

### Checklist

- [ ] If this PR adds a new feature, or fixes a bug in a previously released version, it includes an entry in `NEWS.md` 
- [ ] If this PR adds or changes UI, the updated UI meets [accessibility standards](https://github.com/rstudio/rstudio/wiki/Accessibility)
- [ ] A reviewer is assigned to this PR (if unsure who to assign, check Area Owners list)
- [ ] This PR passes all local unit tests

Signed contributor agreement

### CODE REVIEW

File: src/gwt/src/org/rstudio/studio/client/workbench/commands/CmdConstants_fr.properties [hunk 1]
```diff
@@ -1009,7 +1009,7 @@ shrinkSelectionDesc = Réduire la sélection
 # goToNextSection
 goToNextSectionLabel = Aller à la section suivante
 goToNextSectionButtonLabel = 
-goToNextSectionDesc = Aller à la section/chapitre suivante
+goToNextSectionDesc = Aller à la section/chumk suivant
```
Comment (olivroy) [inline src/gwt/src/org/rstudio/studio/client/workbench/commands/CmdConstants_fr.properties:1012]:
```suggestion
goToNextSectionDesc = Aller à la section/chunk suivant
```

File: src/gwt/src/org/rstudio/studio/client/workbench/commands/CmdConstants_fr.properties [hunk 2]
```diff
@@ -1042,9 +1042,9 @@ openRoxygenQuickReferenceDesc = Référence rapide Roxygen
 
 # toggleDocumentOutline
 toggleDocumentOutlineLabel = Afficher les grandes lignes du document
-toggleDocumentOutlineButtonLabel = le Contour
-toggleDocumentOutlineMenuLabel = Afficher le contour du document
-toggleDocumentOutlineDesc = Afficher le contour du document
+toggleDocumentOutlineButtonLabel = Schéma
+toggleDocumentOutlineMenuLabel = Afficher le schéma du document
+toggleDocumentOutlineDesc = Afficher le schéma du document
```
Comment (olivroy) [inline src/gwt/src/org/rstudio/studio/client/workbench/commands/CmdConstants_fr.properties:1047]:
```suggestion
toggleDocumentOutlineButtonLabel = Plan
toggleDocumentOutlineMenuLabel = Afficher le plan du document
toggleDocumentOutlineDesc = Afficher le plan du document
```

File: src/gwt/src/org/rstudio/studio/client/workbench/commands/CmdConstants_fr.properties [hunk 3]
```diff
@@ -2493,7 +2493,7 @@ clearBackgroundJobsMenuLabel = Effa_cer les travaux locaux
 clearBackgroundJobsDesc = Nettoyer tous les travaux locaux terminés
 
 # runSelectionAsBackgroundJob
-runSelectionAsBackgroundJobMenuLabel = Sélection Ru_n comme travail local
+runSelectionAsBackgroundJobMenuLabel = Selctio_n comme travail local
```
Comment (olivroy) [inline src/gwt/src/org/rstudio/studio/client/workbench/commands/CmdConstants_fr.properties:2496]:
```suggestion
runSelectionAsBackgroundJobMenuLabel = Exéciter la sélectio_n comme travail local
```

File: src/gwt/src/org/rstudio/studio/client/workbench/prefs/PrefsConstants_fr.properties [hunk 4]
```diff
@@ -318,7 +318,7 @@ quartoPreviewLabel=Cette version de RStudio inclut un aperçu de Quarto, un nouv
 enableQuartoPreviewCheckboxLabel=Activer l''aperçu de Quarto
 helpLinkCaption=En savoir plus sur Quarto
 rMarkdownHeaderLabel=R Markdown
-rMarkdownShowLabel=Afficher le contour du document par défaut
+rMarkdownShowLabel=Afficher le schéma du document par défaut
```
Comment (olivroy) [inline src/gwt/src/org/rstudio/studio/client/workbench/prefs/PrefsConstants_fr.properties:321]:
```suggestion
rMarkdownShowLabel=Afficher le plan du document par défaut
```

File: src/gwt/src/org/rstudio/studio/client/workbench/prefs/PrefsConstants_fr.properties [hunk 5]
```diff
@@ -349,7 +349,7 @@ visualModeGeneralCaption=Général
 visualModeUseVisualEditorLabel=Utiliser l''éditeur visuel par défaut pour les nouveaux documents
 visualModeHelpLink=En savoir plus sur le mode d''édition visuelle
 visualModeHeaderLabel=Afficher
-visualEditorShowOutlineLabel=Afficher le contour du document par défaut
+visualEditorShowOutlineLabel=Afficher le schéma du document par défaut
```
Comment (olivroy) [inline src/gwt/src/org/rstudio/studio/client/workbench/prefs/PrefsConstants_fr.properties:352]:
```suggestion
visualEditorShowOutlineLabel=Afficher le plan du document par défaut
```

File: src/gwt/src/org/rstudio/studio/client/workbench/prefs/model/UserStateAccessorConstants_fr.properties [hunk 6]
```diff
@@ -98,7 +98,7 @@ publishAccountDescription= Le (dernier) compte par défaut utilisé pour la publ
 
 # The preferred width, in pixels, of the document outline pane.
 documentOutlineWidthTitle=
-documentOutlineWidthDescription= La largeur préférée, en pixels, du volet de contour du document.
+documentOutlineWidthDescription= La largeur préférée, en pixels, du volet de schéma du document.
```
Comment (olivroy) [inline src/gwt/src/org/rstudio/studio/client/workbench/prefs/model/UserStateAccessorConstants_fr.properties:101]:
```suggestion
documentOutlineWidthDescription= La largeur préférée, en pixels, du volet de plan du document.
```

**Review by gtritchie — APPROVED** [review]
Merci!

**Pull request merged**